R/utils.R

```diff
@@ -0,0 +1 @@
+`%||%` <- function(l, r) if (is.null(l)) r else l
```

Comment:
Can we add tests for this, I don't think the logic is right. Wouldn't it fail on `FALSE %||% TRUE`?